cucumber.yml

```diff
@@ -1,5 +1,14 @@
 <%
 cassandra_version = ENV['CASSANDRA_VERSION'] || '3.0.1'
```

Comment:
This is just a default, as we always specify a C* runtime, this will almost never get used. We're on 3.3.0 now btw.